content/tutorials/r_setup_codingclub_session/index.Rmd

```diff
@@ -31,46 +31,93 @@ remotes::install_github("inbo/inborutils")
 
 Next load the package:
 
-```{r}
+```{r load_package}
 library(inborutils)
 ```
 
 The function takes four arguments with the following default settings:
 
  - `session_date` which is set to the present day
- - `root_dir` which is set to  `"."`
+ - `root_dir` which is set to  the current/project directory `"."`
  - `src_rel_path` which is set to the subdirectory `src`
  - `data_rel_path` which is set to the subdirectory `data`
 
-```{r}
-setup_codingclub_session(
-  session_date = format(Sys.Date(), "%Y%m%d"),
-  root_dir = ".",
-  src_rel_path = "src",
-  data_rel_path = "data"
-)
-```
 
 So when we just run the function like this:
 
-```{r}
+```{r use_default_values}
 setup_codingclub_session()
 ```
 
-We will get the coding club materials for the date of today (if and only if today there is a coding club, otherwise there are no materials) and these materials will be saved in the `src` and `data` folders of the current working directory.
+We will get the coding club materials for the date of today (if and only if today there is a coding club, otherwise there are no materials and nothing will be downloaded) and these materials will be saved in the `src` and `data` folders of the current working directory or project directory if you work within a RStudio project as recommended in the coding club's [getting started](https://inbo.github.io/coding-club/gettingstarted.html). 
+R scripts are downloaded in `src` while datasets are downloaded in `data` using the date of today in the "YYYYMMDD" format as subfolders. Your coding club folder will look like this:
+
+```
+├── src
+    ├── YYYYMMDD
+    │   └── YYYYMMDD_challenges.r
+└── data
+│   ├── YYYYMMDD
+|   │   ├── YYYYYMMDD_filename1.txt
+|   │   ├── YYYYYMMDD_filename2.txt   # if more than one dataset is used
+    │   └── ...
+```
+
+If you want to get the coding club materials for a past date, use the "YYYYMMDD" format, e.g. for material of August 25, 2020: 
 
+```{r change_date}
+setup_codingclub_session("20200225")
+```
 
-If you want to get the coding club materials for a past date, use the "YYYYMMDD" format, e.g.: 
+The subfolders `./src/20200225` and `./data/20200225` are automatically created and files downloaded if not present:
 
-```{r}
-setup_codingclub_session("20200825")
 ```
+├── src
+    ├── 20200225
+    │   ├── 20200225_create_messy_project.r
+    │   ├── 20200225_challenges.r
+    │   └── 20200225_challenges_solutions.r
+└── data
+│   ├── 20200225
+|   │   └── 20200225_urban_gaia_policy.txt
+```
+
+This function is also interactive: before downloading starts a message appears on screen showing the URL of the files to download and the absolute path where R scripts and data will be downloaded, followed by the question: `Do you want to continue? (Y/n)` as shown below:
+
+![example_message](figures/example_message_before_download.png)
```

Comment:
```suggestion
![example_message](images/example_message_before_download.png)
```

I'll see whether I can already commit this in order to test it locally. In other words, it currently went wrong :wink: